packages/frontend/src/features/videos/components/singleVideoComponents/Quiz.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import {NavLink} from 'react-router-dom'
+import {Colors} from '../../../common/styles/constans/Colors'
+import {Device} from '../../../common/styles/constans/Device'
+import {fontWeight} from '../../../common/styles/constans/fontWeight'
+import {FontSize} from '../../../common/styles/constans/FontSize'
+
+const Wrapper = styled.div`
+  margin:25px auto;
+  max-width: 482px;
+  box-shadow: 0px 0px 2px 2px  ${Colors.OPACITY_GRAY};
+  padding: 16px;
+  border-radius:20px;
+  display:flex;
+  flex-direction:column;
+@media ${Device.tablet} {
+  padding:33px;
+}
+`;
+const Title = styled.h3`
+  font-size: 15px;
+  text-align:center;
+  font-weight: ${fontWeight.BOLD};
+  margin-bottom:15px;
+`;
+const Desc = styled.p`
+  max-width:442px;
+  font-size: ${FontSize.SMALL};
+  line-height: 22px;
+  margin-bottom:25px;
+`;
+const Button = styled(NavLink)`
+  text-align:center;
+  text-decoration:none;
+  cursor: pointer;
+  padding:10px 30px;  
+  border-radius:30px;
+  font-size:18px;
+  font-weight:${fontWeight.BOLD};
+  background-color: ${Colors.BLACK};
+  color: ${Colors.WHITE};
+  margin:0 auto;
+`;
+
+const desc = 'In sed tristique anthe, vitae eleifend erat. Curabitur euismod eros nec tincidunt molestie. Donec maximus, nunc ut dapibus aliquet, lacus nisl rhoncus nisi';
```

Comment:
co to? (czemu nie w props to wpada, daj znac na slac czy wiesz jak to poprawic)

Reply:
To rozwiązanie tymczasowe, bo kafelek był już zrobiony, ale nie zmergowany.